docs/data-sources/pubsub_subscription.md

```diff
@@ -39,21 +39,50 @@ data "aws_iam_policy_document" "mypolicy" {
 
 ### Optional
 
-- `aws_sqs` (Attributes) Encore provisioned SQS Queue information (see [below for nested schema](#nestedatt--aws_sqs))
 - `env` (String) The environment of the Encore resource. Defaults to the provider environment
-- `gcp_pubsub` (Attributes) Encore provisioned PubSub Subscription information (see [below for nested schema](#nestedatt--gcp_pubsub))
+
+### Read-Only
+
+- `aws_sqs` (Attributes) (see [below for nested schema](#nestedatt--aws_sqs))
+- `gcp_pubsub` (Attributes) (see [below for nested schema](#nestedatt--gcp_pubsub))
 
 <a id="nestedatt--aws_sqs"></a>
 ### Nested Schema for `aws_sqs`
 
 Read-Only:
 
 - `arn` (String) The [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for this sqs queue
+- `topic` (Attributes) (see [below for nested schema](#nestedatt--aws_sqs--topic))
+
+<a id="nestedatt--aws_sqs--topic"></a>
+### Nested Schema for `aws_sqs.topic`
+
+Read-Only:
+
+- `arn` (String) The [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for this  sns topic
+
 
 
 <a id="nestedatt--gcp_pubsub"></a>
 ### Nested Schema for `gcp_pubsub`
 
 Read-Only:
 
+- `dlq` (Attributes) (see [below for nested schema](#nestedatt--gcp_pubsub--dlq))
```

Comment:
Do people know what `dlq` stands for, or should we spell it out? (And in the context of GCP it's called a dead-letter topic, not a dead-letter queue)

Reply:
Let's just call it deadletter then